### PR TITLE
Add release builds

### PR DESCRIPTION
**Pull request by sigmaSd**
This pr add automatic releases by github actions.

steps:

- add tag: `git tag -a x.x.x`
- push the new tag: `git push --follow-tags` or `git push origin x.x.x`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+on:
+  push:
+    # Enable when testing release infrastructure on a branch.
+   #branches:
+   #  - master
+  #pull_request:
+  #  branches:
+   #   - master
+   tags:
+    - '[0-9]+.[0-9]+.[0-9]+'
```
Comment (lemunozm) [inline .github/workflows/release.yml]: Could it contain a 'v' before the numbers? To keep parallelism with the previous tag :)

**Review by lemunozm — CHANGES_REQUESTED** [review]

**Comment by lemunozm** [comment]
I like so much this workflow!!

Only a question, it works also with unannotated tags?

**Comment by sigmaSd** [comment]
Yes I just tested with unannotated tags and it works.

File: .github/workflows/release.yml (hunk 1 above)
Comment (sigmaSd) [reply .github/workflows/release.yml]: done

**Pull request merged**